renderer-macos/lib/Brisk.re

```diff
@@ -1,9 +1,8 @@
 open Brisk_core;
-open Cocoa;
 
 module NativeCocoa = {
   [@deriving (show({with_path: false}), eq)]
-  type hostElement = CocoaClass.view;
+  type hostElement = BriskCocoa.view;
```

Comment:
I believe we could have left Cocoa as is. `BriskBindings` make sense because the classes are prefixed but for Cocoa we just have bindings to some stock Cocoa stuff.

Reply:
I would say the same about Foundation.

Reply:
I don't follow, `Foundation`?

We departed from 'Cocoa bindings' to 'stubs for a subset of methods in Cocoa', which is why I'm generally trying to avoid using 'flashy' names like `Cocoa`.

Reply:
It sounded dumb, because we use a Cocoa wrapper module for top-level, but somehow it felt appropriate there.

Reply:
Hmm, I see, i'm not sure though. There has to be some balance. Anyway, in this case I'd just rename the file to Types. We have a Cocoa module anyway.

Reply:
`CocoaTypes`?

Reply:
We do not expose `BriskCocoa` to the outside, should we?

Reply:
Probably not, but we have to show the types to the outside (one way or another)

Reply:
Just pushed some updates, what do you think?

We'll need to add some nice api for `Application`, but I believe overall looks much better now.